cdap-hbase-compat-0.96/pom.xml

```diff
@@ -358,6 +358,21 @@
         </plugins>
       </build>
     </profile>
+    <profile> 
```

Comment:
The problem with that is if you skip the tests then the compat tests will run. Modified the approach to use just a profile to skip.